C-Plus-Plus/cp/Three_Sum.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+Description :
+     Given an array and a value, find if there exists three
+     number whose sum is equal to the given value.
+     If found then output will be true else false.
+
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main()
```

Comment:
Push all the main code logic in a separate function and use main() for driver code